### DISCUSSION TITLE
I need help with changing vehicles by the AI

### DISCUSSION THREAD

**Discussion by Zeta1311** [Vdauphin/HeartsAndMinds#1514] [Question and answer]
I want to add other vehicles and stations, but I can't find the option to do it. I would like some help :c

**Comment by Vdauphin** [comment]
Hello,
Look here https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/mission.sqf#L645-L666
Cheers
